packages/components/src/color-picker/component.tsx

```diff
@@ -49,8 +49,17 @@ const UnconnectedColorPicker = (
 		onChange,
 		defaultValue = '#fff',
 		copyFormat,
+
+		// Context
+		onPickerDragStart,
+		onPickerDragEnd,
```

Comment:
These "props" are only exposed via internal context, therefore not causing an API change to the component